src/sentry/static/sentry/app/views/settings/organizationGeneralSettings/organizationSettingsForm.jsx

```diff
@@ -48,14 +63,28 @@ class OrganizationSettingsForm extends AsyncComponent {
         onSubmitError={() => addErrorMessage('Unable to save change')}
       >
         <PermissionAlert />
-        <JsonForm
-          additionalFieldProps={{hasSsoEnabled: !!authProvider}}
-          features={new Set(organization.features)}
-          access={access}
-          location={this.props.location}
-          forms={organizationSettingsFields}
-          disabled={!access.has('org:write')}
-        />
+        <JsonForm {...jsonFormSettings} forms={organizationSettingsFields} />
+
+        <Feature features={['datascrubbers-v2']}>
+          {({hasFeature}) =>
+            hasFeature ? (
+              <Panel>
+                <PanelHeader>{t('Security & Privacy')}</PanelHeader>
+                <EmptyMessage
+                  title={t('Security & Privacy section now has its own tab')}
+                  description={
+                    <Link to={`/settings/${orgId}/security-and-privacy/`}>
+                      {t('Go to Security & Privacy')}
+                    </Link>
+                  }
+                />
+              </Panel>
+            ) : (
+              <JsonForm {...jsonFormSettings} forms={organizationSecurityAndPrivacy} />
```

Comment:
Ah I missed this, this looks like where we display the form if you don't have the feature.

Reply:
Yeah, so for context we're moving both project and org settings related to data privacy and security to their own tab such that we can later add more settings without overloading the already overloaded general settings. For non-alpha customers we don't intend to move or add anything at all.